chart/values.yaml

```diff
@@ -704,6 +718,51 @@ scheduler:
     # Annotations to add to scheduler kubernetes service account.
     annotations: {}
 
+
+  # Allow KEDA autoscaling.
+  keda:
+    enabled: false
+    namespaceLabels: {}
+
+    # How often KEDA polls the airflow DB to report new scale requests to the HPA
+    pollingInterval: 5
+
+    # How many seconds KEDA will wait before scaling to zero.
+    # Note that HPA has a separate cooldown period for scale-downs
+    cooldownPeriod: 30
+
+    # Minimum number of workers created by keda
+    minReplicaCount: 0
+
+    # Maximum number of workers created by keda
+    maxReplicaCount: 10
+
+    # Specify HPA related options
+    advanced: {}
+    # horizontalPodAutoscalerConfig:
+    #   behavior:
+    #     scaleDown:
+    #       stabilizationWindowSeconds: 300
+    #       policies:
+    #         - type: Percent
+    #           value: 100
+    #           periodSeconds: 15
+
+    # Specify the KEDA Triggers for autoscaling
+    triggers: >-
+      - type: postgresql
+        metadata:
+          targetQueryValue: "1"
+          connectionFromEnv: AIRFLOW_CONN_AIRFLOW_DB
+          query: >-
+            SELECT ceil(COUNT(*)::decimal / {{ .Values.config.core.parallelism }})
+            FROM task_instance
+            WHERE (state='running' OR state='queued')
+            {{- if eq .Values.executor "CeleryKubernetesExecutor" }}
+            AND queue != '{{ .Values.config.celery_kubernetes_executor.kubernetes_queue }}'
+            {{- end }}
```

Comment:
why would this query be a good metric for scaling triggerer?

Reply:
The worker query makes sense because all tasks need workers (except those running on kubernetes hence the case for that)

However, not all tasks are deferrable.  So you want to look not at overall queued tasks, but tasks i think in a deferred state which have not been picked up by a triggerer.  Then you'd want to compare that with triggerer concurrency not overall parallelism I believe.

Reply:
@dstandish perhaps this topic should be on a separate conversation? 
our company atm is using a semi-hacked way of this (we basically set replicas to `~` and then manually defined the same definitions I've added here). I will add some test cases over the weekend (or monday) and after that it would be nice to merge this :)

In the meantime maybe we should transfer this to a github issue on how to improve the queries? I feel like it should be ok, since my change allows people to set whatever query they want.

Reply:
@asosnovsky i don't think so. i don't think the query needs to be perfect.  but at least it shouldn't be completely non-sensical, which is the case right now.  the query should have some basic justifiability even if it's not perfect.  don't you think?  otherwise users will naively use this feature thinking it has a sensible default and find that the scaling has literally nothing to do with how many triggers are running.